app/controllers/inthenewscommentaries_controller.rb

```diff
@@ -13,6 +13,7 @@ def home
     }).facets
 
     set_page_facets
+    @featured = Commentary.all.sample(1).first
```

Comment:
Looks like this can be written as `@featured = Commentary.all.sample` to get the same outcome: https://ruby-doc.org/core-2.6.6/Array.html#method-i-sample